### DISCUSSION TITLE
Queue System

### DISCUSSION THREAD

**Discussion by Chicalho** [lllyasviel/Fooocus#1886] [Ideas]
One of the most common explorations I find myself doing is using the same prompt with different styles and scales to better understand how each one influences the output as they are mixed and matched.

The UI for generating output with a "single configuration" is awesome. But when you want to generate 1 output for each configuration based on the same prompt, you have to wait for the render until you can request a new one.

IDEA:
Create a queue system (like the one in Handbrake for example) so we can queue multiple requests and let the computer process them in order during the night or any other off period without manually hitting "generate" each time.

Love the software,
Francisco

**Comment by mashb1t** [comment]
There is a version in development, but it's not compatible for multi-user scenarios yet.
You can find it here: https://github.com/lllyasviel/Fooocus/pull/1773
